### PR TITLE
added more info/questions for php section

### PR DESCRIPTION
**Pull request by batusa**
- Is your code-style following PSR standards?
- Does your team try hard to follow good design principles such as: SOLID, DRY, Object calisthenics?
- Do you encourage your developers to take the ZCE exam?

### CODE REVIEW

**Comment by ChiperSoft** [comment]
The design principles question would probably be better under the Development Process section. I could move that post-merge or you could do it as part of the PR, which would you prefer?

**Comment by ChiperSoft** [comment]
Actually, I think I want to reword that question, so I'll move it myself.  Thanks for the contributions :+1:

**Comment by batusa** [comment]
you're welcome :+1:
